lua/mellifluous/highlights/plugins/lazy.lua

```diff
@@ -3,11 +3,13 @@ local M = {}
 function M.set(hl, colors)
     local config = require("mellifluous.config").config
     local groups = require("mellifluous.highlights.custom_groups").get(colors)
-    local bg = hl.get("NormalFloat").bg
+
+    local f_bg = hl.get("NormalFloat").bg
+    local bg_active = (type(f_bg) == "string" and f_bg == "NONE") and colors.bg3 or f_bg
```

Comment:
I'm assigning the background value to `bg_active` instead of `bg` to make it easier to rebase #49 on top of this PR (if you ever want to accept it), since the two are closely related.